GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1902,6 +1903,18 @@ private void ToolStripButtonPushClick(object sender, EventArgs e)
             PushToolStripMenuItemClick(sender, e);
         }
 
+        private void ToolStripFileExplorerClick(object sender, EventArgs e)
```

Comment:
there is already a handler doing just this - `FileExplorerToolStripMenuItemClick`
please bind the event to the existing handler